Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml

```diff
@@ -0,0 +1,338 @@
+  value: {}
+- description: Remediation Guidance of the Attack Surface Rule.
+  key: RemediationGuidance
+  playbookInputQuery:
+  required: true
+  value: {}
+name: Cortex ASM - Instant Message
+outputSections:
+- description: Generic group for outputs
+  name: General (Outputs group)
+  outputs: []
+outputs: []
+starttaskid: "0"
+tasks:
+  "0":
+    continueonerrortype: ""
+    id: "0"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "11"
+      - "13"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 18a8542a-f6ca-474e-82bb-cd023bdf2a69
+      iscommand: false
+      name: ""
+      version: -1
+      description: ''
+    taskid: 18a8542a-f6ca-474e-82bb-cd023bdf2a69
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -170
+        }
+      }
+  "5":
+    continueonerrortype: ""
+    id: "5"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: e28fdfca-d524-4853-8dc3-285981f0050f
+      iscommand: false
+      name: Complete
+      type: title
+      version: -1
+      description: ''
+    taskid: e28fdfca-d524-4853-8dc3-285981f0050f
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 520
+        }
+      }
+  "8":
+    continueonerrortype: ""
+    id: "8"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "5"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmnotification
+      keys:
+        simple: type,value,url,timestamp
+      val1:
+        simple: Slack
+      val2:
+        simple: ${Slack.Thread.ID}
+      val3:
+        simple: n/a
+      val4:
+        simple: TIMESTAMP
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: |-
+        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. For example:
+        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
+      id: e529a3dc-af23-41e5-8445-64473ce4b094
+      iscommand: false
+      name: Set notification grid field
+      script: GridFieldSetup
+      type: regular
+      version: -1
+    taskid: e529a3dc-af23-41e5-8445-64473ce4b094
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 360
+        }
+      }
+  "10":
+    continueonerrortype: ""
```

Comment:
What should happen if the slack message fails to send?

Reply:
I would saw that a failures should be good enough (what we do for other notifications), but I am open for other options